### PR TITLE
 ✅ eds-utils tests

### LINKED ISSUE #1868: Add tests for eds-utils
Add test for remaining hooks and utils.

### PR DESCRIPTION
**Pull request by oddvernes**
Resolves #1868

### CODE REVIEW

File: packages/eds-utils/src/hooks/useHideBodyScroll.test.tsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  return <div></div>
+}
+
+describe('useHideBodyScroll', () => {
+  it('Should add "overflow: hidden" on document.body only while input is true', () => {
+    const { rerender } = render(<TestComponent hideScroll={true} />)
+    expect(document.body.style.overflow).toBe('hidden')
+    rerender(<TestComponent hideScroll={false} />)
+    expect(document.body.style.overflow).toBe('')
+  })
```
Comment (mimarz) [inline packages/eds-utils/src/hooks/useHideBodyScroll.test.tsx:15]: Maybe add a test where body overflow is set to some other value and that its re-instated after `hideScroll` has been triggered?

File: packages/eds-utils/src/hooks/useHideBodyScroll.ts [hunk 2]
```diff
@@ -1,15 +1,15 @@
 import { useEffect, useRef } from 'react'
 
 export const useHideBodyScroll = (active: boolean): void => {
-  const overflowState = useRef<string | undefined>(document.body.style.overflow)
+  const overflowState = useRef<string | undefined>()
```
Comment (mimarz) [inline packages/eds-utils/src/hooks/useHideBodyScroll.ts:4]: Did we find another bug? 🙈

File: packages/eds-utils/src/hooks/usePopper.test.tsx [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    const content = screen.getByText('content')
+
+    expect(menu).not.toHaveAttribute('data-popper-placement')
+    fireEvent.click(button)
+
+    await waitFor(() => {
+      expect(menu).toHaveAttribute('data-popper-placement', 'top')
+      expect(content).toHaveAttribute('style')
+    })
+  })
```
Comment (mimarz) [inline packages/eds-utils/src/hooks/usePopper.test.tsx:54]: Maybe add one test for checking if placement and offset is properly applied/forwarded?

**Review by mimarz — CHANGES_REQUESTED** [review]
Good work! Added some suggestion on some additional tests apart from that it looks good :)

File: packages/eds-utils/src/hooks/useHideBodyScroll.test.tsx (hunk 1 above)
Comment (oddvernes) [reply packages/eds-utils/src/hooks/useHideBodyScroll.test.tsx:15]: good point 👍

File: packages/eds-utils/src/hooks/useHideBodyScroll.ts (hunk 2 above)
Comment (mimarz) [reply packages/eds-utils/src/hooks/useHideBodyScroll.ts:4]: Oh, right. Good thing we are getting this streamlined.

File: packages/eds-utils/src/hooks/useHideBodyScroll.test.tsx (hunk 1 above)
Comment (oddvernes) [reply packages/eds-utils/src/hooks/useHideBodyScroll.test.tsx:16]: Done

File: packages/eds-utils/src/hooks/usePopper.test.tsx (hunk 3 above)
Comment (oddvernes) [reply packages/eds-utils/src/hooks/usePopper.test.tsx:77]: Added test for placement and offset 👍

**Review by mimarz — APPROVED** [review]
LGTM 👍
